iocBoot/ioc-mfx-motion/Makefile

```
@@ -1,18 +1,25 @@
-IOC_TOP=/reg/g/pcds/epics/ioc/common/ads-ioc/R0.5.0
+IOC_TOP=/reg/g/pcds/epics/ioc/common/ads-ioc/R0.6.0
 IOC_INSTANCE_PATH := $(shell pwd)
 
 # Set PRODUCTION_IOC to 1 to move from a testing to a production IOC:
-PRODUCTION_IOC := 1
+PRODUCTION_IOC := 0
```

```suggestion
PRODUCTION_IOC := 1
```

And don't forget to flip this back